x-pack/plugins/observability_ai_assistant/server/functions/esql/system_message.txt

```diff
@@ -170,7 +178,7 @@ Here are some example queries:
 ```esql
 FROM employees
 | WHERE still_hired == true
-| EVAL hired = DATE_FORMAT("YYYY", hire_date)
+| EVAL hired = DATE_FORMAT("MMMM ' from ' YYYY", hire_date) // add words to the date format using single quotes ''
```

Comment:
is this necessary? or was this because of the incorrect formatting?

Reply:
it was because of the incorrect formatting, but as that is fixed i'll revert it